diff --git a/fireshape/control.py b/fireshape/control.py
index 0ed33f9..d01ada8 100644
--- a/fireshape/control.py
+++ b/fireshape/control.py
@@ -153,8 +153,9 @@ class FeControlSpace(ControlSpace):
         self.mesh_r = mesh_r
         element = self.mesh_r.coordinates.function_space().ufl_element()
         family = element.family()
-        degree = element.degree() + add_to_degree_r
-        self.V_r = fd.VectorFunctionSpace(self.mesh_r, family, degree)
+        # degree = element.degree() + add_to_degree_r
+        # self.V_r = fd.VectorFunctionSpace(self.mesh_r, family, degree)
+        self.V_r = fd.VectorFunctionSpace(self.mesh_r, 'CG', 1)
         self.V_r_dual = self.V_r.dual()
 
         # Create self.id and self.T, self.mesh_m, and self.V_m.
@@ -163,7 +164,8 @@ class FeControlSpace(ControlSpace):
         self.T = fd.Function(self.V_r, name="T")
         self.T.assign(self.id)
         self.mesh_m = fd.Mesh(self.T)
-        self.V_m = fd.VectorFunctionSpace(self.mesh_m, family, degree)
+        # self.V_m = fd.VectorFunctionSpace(self.mesh_m, family, degree)
+        self.V_m = fd.VectorFunctionSpace(self.mesh_m, 'CG', 1)
         self.V_m_dual = self.V_m.dual()
 
         if mesh_c is not None:
